specification/resources/resource-manager/Microsoft.Authorization/2016-12-01/policyDefinitions.json

```diff
@@ -279,8 +295,32 @@
         },
         "x-ms-pageable": {
           "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.Authorization/policydefinitions": {
```

Comment:
was this api missing from the spec before but existed in the service?

Reply:
yes, we never added this to spec